ui/imports/shared/views/chat/CompactMessageView.qml

```diff
@@ -72,6 +72,14 @@ Item {
     height: messageContainer.height + messageContainer.anchors.topMargin
             + (dateGroupLbl.visible ? dateGroupLbl.height + dateGroupLbl.anchors.topMargin : 0)
 
+    Connections {
+        target: root.messageStore.messageModule
```

Comment:
You can add `enabled: responseTo !== ""` since it's useless to check this signal when this is not a reply